client/knowledge_base_search/src/containers/ResultContainer/ResultContainer.jsx

```diff
@@ -18,6 +45,9 @@ function ResultContainer({ resultText, resultRank, children }) {
           <div className="result_container_text--div">
             { resultText }
           </div>
+          <div className="training_comparison--content_rank">
```

Comment:
i would pass this stuff in as the `children` and move all the logic for training back to the `TrainingComparison` class due to the fact that this class is re-used by both passages and training. Right now, passages will have an extra `div` for no reason. If we need different result containers per feature, we should split this out accordingly